Library/ViewModels/LiveStreamEventDetailsViewModel.swift

```diff
@@ -101,7 +104,7 @@ public final class LiveStreamEventDetailsViewModel: LiveStreamEventDetailsViewMo
 
     let eventEvent = configData
       .switchMap { project, liveStream, optionalEvent in
-        fetchEvent(forProject: project, liveStream: liveStream, event: optionalEvent)
+        fetchEvent(forProjectLiveStreamData: (project, liveStream: liveStream, event: optionalEvent))
           .materialize()
     }
 
```

Comment:
you can condense line 182 below (`configData.filter { _, _, event in event == nil }.mapConst(true)`) to:

`configData.filter { $0.event == nil }.mapConst(true)`